lib/phlex/rails/helpers.rb

```diff
@@ -1,6 +1,14 @@
 # frozen_string_literal: true
 
 module Phlex::Rails::Helpers
+	def self.helpers
+		constants.map { |constant| const_get constant }
+	end
+
+	def self.included(component)
+		helpers.each { |helper| component.include helper }
+	end
```

Comment:
Rather than include each helper on the included hook, I think this Helpers module itself could include all the helpers just once.